### PR TITLE
CEO#4.10 - CEO Card Rendering + PlayerHome.vue

### PR DESCRIPTION
**Pull request by d-little**
### Card Rendering

A few things in this change.  

##### [assets/misc/arrow_opg.png → assets/ceos/arrow_opg.png](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:ceoRenderCard?expand=1#diff-960a6e0d99b8600ebe474caba8b8e08797afa7143c02a9eb45860a4a10eaf695)
I've moved the OPG assets that already existed from ./assets/misc/ to ./assets/ceo/

##### [assets/ceos/arrow_opg_dark.png](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:ceoRenderCard?expand=1#diff-463b0a77e90722b422e4accc38ff1016616429d2e22aa088d93d5186b15ef90f)

A 'dark' version of OPG that we can use when the card has been used. (Not implemented.. yet)

##### [src/client/components/PlayerHome.vue](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:ceoRenderCard?expand=1#diff-8e4aa1d7e2abadada4d84dd026c3986585af12eb0aa20f2a822572c3d49e04de)

Hopefully all self explanatory

##### [src/client/components/card/Card.vue](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:ceoRenderCard?expand=1#diff-919053deacaf8c365208787e2d2b232582f7c7ad1a2997ff9763ea023c37d7bd)

CEO cards have no cost, this is just including it with the Corporations + Preludes

##### [src/styles/cards.less](https://github.com/terraforming-mars/terraforming-mars/compare/main...d-little:ceoRenderCard?expand=1#diff-ad5c16d1a5a2c20b9e4cba899c81d55297fe0e9ea9556e78aedf1169ac9375ea)
I've made the CEO card icon a cool little briefcase... Just because.  We can make this a C.

##### Everything Else

I hope all the rest just makes sense.

### CODE REVIEW

**Comment by d-little** [comment]
I've messed up the rendering of all card prices.. need to see what I've done

**Review by kberg — COMMENTED** [review]
Rename icon_leaders?

File: src/client/components/card/Card.vue [hunk 1]
```diff
@@ -102,12 +102,12 @@ export default Vue.extend({
     getCost(): number | undefined {
       const cost = this.cardInstance.cost;
       const type = this.getCardType();
-      return cost === undefined || type === CardType.PRELUDE || type === CardType.CORPORATION ? undefined : cost;
+      return type === CardType.PRELUDE || type === CardType.CORPORATION || type === CardType.CEO ? undefined : cost;
```
Comment (kberg) [inline src/client/components/card/Card.vue:105]: This line and line 110 had a rhyming cadence, can you restore them?

**Review by kberg — APPROVED** [review]

File: src/client/components/card/Card.vue (hunk 1 above)
Comment (d-little) [reply src/client/components/card/Card.vue]: I accidentally removed that cost === defined, missed it on commit

I reaaaaally dont like long lines like this, so sometimes I try to fix them and then realise that's the python in me and put up with 100+ character lines :(

**Review by kberg — APPROVED** [review]

**Pull request merged**